src/test/fuzz/addrman.cpp

```diff
@@ -26,7 +26,7 @@ FUZZ_TARGET_INIT(addrman, initialize_addrman)
 {
     FuzzedDataProvider fuzzed_data_provider(buffer.data(), buffer.size());
     SetMockTime(ConsumeTime(fuzzed_data_provider));
-    CAddrMan addr_man(/* deterministic */ true);
+    CAddrMan addr_man(/* deterministic */ true, /* consistency_checks */ 0);
```

Comment:
Could it make sense to enable the consistency checks here? 

I saw the conversation here https://github.com/bitcoin/bitcoin/pull/20233#issuecomment-755188580, where you said: 
> I think somewhere that they're definitely useful would be in fuzz testing, to ensure that there's no way to violate addrman's invariants.

So wondering why you landed on disabling.

Reply:
Added here and to the other fuzzers.